backend/habragasolina/src/main/java/bo/edu/usfa/gasolina/habragasolina/Service/UserService.java

```diff
@@ -26,51 +22,32 @@ public UserService(UserRepository userRepository, GasStationRepository gasStatio
     
     public User saveUser(User user) {
 
-        //validations
-        if(user.getPassword().length() <= 6){
-            throw new ResponseStatusException(HttpStatus.NOT_FOUND, "Password must be longer or equal than 6 characters");
-        }
-        
-        if(userRepository.existsByUsername(user.getUsername())){
-            throw new ResponseStatusException(HttpStatus.BAD_REQUEST, "User is duplicated");
+        if(user.getUsername().length() <= 3){
+            throw new ResponseStatusException(HttpStatus.BAD_REQUEST, "UserName must be longer than 3 characters");
         }
 
-        if(user.getUsername().length() <= 3){
-            throw new ResponseStatusException(HttpStatus.NOT_FOUND, "UserName must be longer than 3 characters");
+        if(user.getPassword().length() <= 6){
```

Comment:
If i send a password that is 6 characters long it will fail? So the message is not correct or the validation is not correct. Same as the user name

Reply:
I'll fix the password validation, although I believe that the username validation is correct as it is because it is stated in the Jira CreateUser that "UserName must be longer than 3 characters", and not "at least or equal than 3 characters", so if the user tries to post a 3 character username, they should get a validation error